tests/system/providers/amazon/aws/example_redshift.py

```diff
@@ -177,13 +177,6 @@ def delete_security_group(sec_group_id: str, sec_group_name: str):
         cluster_identifier=redshift_cluster_identifier,
     )
 
-    resume_cluster_in_deferred_mode = RedshiftResumeClusterOperator(
-        task_id="resume_cluster_in_deferred_mode",
-        cluster_identifier=redshift_cluster_identifier,
-        deferrable=True,
-    )
-    # [END howto_operator_redshift_resume_cluster]
```

Comment:
I think this END tag needs to stay, doesn't it?

Reply:
yep, my mistake. Thanks